lib/createDist.js

```diff
@@ -3,19 +3,16 @@ const util = require('../lib/util')
 const path = require('path')
 const fs = require('fs-extra')
 
-const createDist = (options) => {
+const createDist = (buildConfig = config.defaultBuildConfig, options) => {
+  config.buildConfig = buildConfig
```

Comment:
I'm not sure I understand this change. `config.update` should handle updating the internal config and I don't think it's a good idea to set `buildConfig` directly like this. What is the reason for this change?

Reply:
I see, it's just matching `build`